src/docs/content/utils/shared.mdx

```diff
@@ -0,0 +1,57 @@
+import {
+  Tabs,
+  TabsContent,
+  TabsList,
+  TabsTrigger,
+} from "@components/Tabs/Tabs";
+import { DynamicCode } from "./DynamicCode";
+import {
+  ComponentDemo,
+  ComponentSourceTabs,
+  ComponentExamples, DocPre, MdxP
+} from "src/docs/components/MdxComponents";
+
+
+# {props.title}
+
+<MdxP>{props.description}</MdxP>
+
+<ComponentDemo type={props.slug} />
+
+## Installation
+
+If you haven't initialized MailingUI in your project, please refer to [Installation](/docs/installation) docs first.
+
+Choose an a) automatic install by running our `@mailingui/cli` following the commands/options provided or b) install manually by copying and pasting directly into your `src/mailingui/components` directory.
+
+  <Tabs defaultValue="automatic">
+    <div className="flex items-center justify-start w-full mb-8">
+      <TabsList className="mt-8 rounded-2xl">
+        <TabsTrigger value="automatic" className="px-7 py-2 rounded-[10px]">Automatic Install</TabsTrigger>
+        <TabsTrigger value="manual" className="px-7 py-2 rounded-[10px]">Manual Install</TabsTrigger>
+      </TabsList>
+    </div>
+
+    <TabsContent value="automatic">
+      <DynamicCode replaceBy={props.installName}>
+        ```shell
+        npx @mailingui/cli add [options] installName
```

Comment:
Imho much better, I would probably also remove [options] in sake of runnable copy code, but clarity approach also fine to me.

Reply:
I read this comment after I pushed the changes, so I left it as is for now. Let me know if you'd like to change it